### DISCUSSION TITLE
optional default-Browser selection

### DISCUSSION THREAD

**Discussion by Gamer1986PAN** [lllyasviel/Fooocus#3117] [Ideas]
Would it be possible to add an option to change the default browser this program uses? I use firefox as my main browser but i would like to seperate this program using chrome to minimize the risk of firefox getting an error. Is there already an option i just did not find or could something like that be added?

**Comment by Rogala** [comment]
After starting the console, launch any browser and type in the address:
`http://127.0.0.1:7865/`

**Reply by Gamer1986PAN** [reply]
Yeah i have done that but it still default opens firefox whenever i have to restart the program (my pc specs are bare minimum right now for the software to run especially when i have a game GPU based running at the same time)

**Reply by mashb1t** [reply]
this depends on your system and gradio doesn't have a browser selection, sorry
